tests/parser.rs

```diff
@@ -19,10 +19,10 @@ use joker::track::Untrack;
 use serde_json::value::Value;
 use std::ffi::OsStr;
 use std::fs::{File, read_dir};
-use std::io::Read;
+use std::io::{Read, Write, stdout};
 use std::path::Path;
 use std::{thread, env};
-use test::{TestDesc, TestDescAndFn, TestName, TestFn, test_main};
+use test::{TestDesc, TestDescAndFn, TestName, TestFn, Bencher, TDynBenchFn, test_main};
```

Comment:
Do you mind adding a comment to the `extern crate test` line above that explains that the `rustc-test` crate brought in by Cargo.toml sets its crate name to `test`? Since this is unconventional it took me forever to figure out where these API's were coming from.